getDateFromBancAccaunt/account_spec.rb

```diff
@@ -7,37 +7,54 @@
 
 require_relative 'example_bank'
 require 'rspec'
+
+def copyhash(inputhash)
+  h = Hash.new
+  inputhash.each_with_index do |pair, index|
+    h.store(pair[0], pair[1])
+    if index > 2
+      break
+    end
+  end
+  return h
+end
+
 str = IO.read("temp.json")
 my_hash = JSON.parse(str)
 accounts = []
-my_hash['account'].each do |item|
-  accounts << Account.new(
-    item['name'],
-    item['currency'],
-    item['balance'].to_f,
-    item['nature']
-  )
+my_hash['accounts'].each do |item|
+  accounts << copyhash(item)
 end
+
 example_bank = ExampleBank.new
 describe 'accounts' do
   it 'should receive 5 for accounts' do
-    example_bank.connect
-    example_bank.fetch_accounts
+    html_example = Nokogiri::HTML(File.read('account/index0.html'))
+    strct = Nokogiri::HTML(html_example).css('script[id="data"]')
+    example_bank.parse_accounts(strct)
     expect(example_bank.accounts.count).to eq(5)
```

Comment:
Could you also display the results.

As in this example for an Account object:
expect(accounts.first.to_hash).to eq(
    {
      "name"         => "Demo Everyday Account",
      "currency"     => "USD",
      "balance"      => 1959.90,
      "nature"       => "account",
      "transactions" => []
    }
  )

Same for Transactions object:
expect(transactions.first.to_hash).to eq(
  {
    "date": "2020-10-07",
    "description": "bought food",
    "amount": -89.31,
    "currency": "USD",
    "account_name": "Demo Everyday Account"
  }
)